### ISSUE TITLE
Water doesn't flow properly in some situations

### ISSUE THREAD

**Issue by supersnoro** [StrangeLoopGames/EcoIssues#11306]
![image](https://user-images.githubusercontent.com/4164893/52762491-d9d2d500-2fe5-11e9-8641-729e39b31c25.png)
all 3 water pumps are working and powered with 2 waterwheels, you can see the water spill behind the pipes
![image](https://user-images.githubusercontent.com/4164893/52762498-ea834b00-2fe5-11e9-9b64-04f3c8dfb9f8.png)
liquid flow is only 0.1 liters
![image](https://user-images.githubusercontent.com/4164893/52762510-f707a380-2fe5-11e9-8232-a6a12fd68e74.png)
turning off the 2 water pumps to the right gets rid of the water spilling animation, turning either one back on makes it come back
then I delete the pipe connecting to the pump to the left, as such:
![image](https://user-images.githubusercontent.com/4164893/52762523-0555bf80-2fe6-11e9-9423-25c1b50312b7.png)
![image](https://user-images.githubusercontent.com/4164893/52762526-0981dd00-2fe6-11e9-8afb-46b46c7ee88a.png)
flow is now 0.2
![image](https://user-images.githubusercontent.com/4164893/52762534-130b4500-2fe6-11e9-8be3-d6542fdd829e.png)
adding a pump on the other side & linking it with a pipe increases flow to 0.3
![image](https://user-images.githubusercontent.com/4164893/52762559-2e765000-2fe6-11e9-8ba4-228c841eb54e.png)
![image](https://user-images.githubusercontent.com/4164893/52762579-3c2bd580-2fe6-11e9-8e0c-46396454a887.png)
trying the same setup on the other side doesn't work, there's a water spill on the left again (also, not sure what's up with the leftmost pipe, maybe it's linked to the issue?)
![image](https://user-images.githubusercontent.com/4164893/52762601-4a79f180-2fe6-11e9-86b8-be537beb4863.png)
only considers one water pump with the 3 pumps on the left
![image](https://user-images.githubusercontent.com/4164893/52762614-56fe4a00-2fe6-11e9-944e-c7a6b4c069a4.png)
turned the 2 water pumps on the right off and the spill on the left disappeared - turning either back on makes the spill come back
Not sure it matters, but the setup that doesn't work is to the west in this creative world. I'll validate which direction the pipes were on the server I encountered the issue in once it gets updated to the new version.

EDIT: This is in the 0.8.0.2 beta build
EDIT2: I just checked on the server I've been playing on, and the part of the setup that's not working was also facing west.

**Comment by johnkslg** [comment]
@supersnoro thanks for the very detailed description! Can you send me your game.eco file with coords of the problem?

**Comment by supersnoro** [comment]
Oops, sorry! I forgot to reply after I saw your message.

No problem for the description! I know first hand how frustrating it can be to have some information missing, so I tried to include as much information as I could! Here are the .eco and .db files for the world I reproduced this in. The setup is at 697,61,710. It should already be in the bugged setup.

[Storage.zip](https://github.com/StrangeLoopGames/EcoIssues/files/2922419/Storage.zip)
